src/thriveopedia/pages/ThriveopediaMuseumPage.tscn

```diff
@@ -175,5 +190,16 @@ margin_bottom = 599.0
 rect_min_size = Vector2( 391, 0 )
 WindowTitle = "CONFIRM_EXIT"
 
-[connection signal="pressed" from="HBoxContainer/SpeciesDetailPanel/MarginContainer/VBoxContainer/OpenInFreebuildButton" to="." method="OnOpenInFreebuildPressed"]
+[node name="NoDirectoryDialog" parent="." index="3" instance=ExtResource( 8 )]
+margin_left = 340.0
+margin_top = 360.0
+margin_right = 940.0
+margin_bottom = 360.0
+rect_min_size = Vector2( 600, 0 )
```

Comment:
The dialog box is too wide, I think lowering this to 300 would be good.

Reply:
Done. It does look better.